### DISCUSSION TITLE
Tutorial Question: Platformer - How can I make a character climb a wall

### DISCUSSION THREAD

**Discussion by jdiperla** [replit/kaboom#640] [Q&A]
Hi there,

I am not having trouble with the character moving or anything. But from time to time in my level, I want the character to scale a wall. I am just not sure what the best way to go about this would be. Any suggestions?

**Comment by kgish** [comment]
If he is touching the wall (ladder or whatever) turn off gravity and make the up key climb, animation plus dx--.
